names/jacob_conrad.txt

```diff
@@ -0,0 +1,4 @@
+Jacob Conrad
+footall.jpg
```

Comment:
There is a typo in your image file -- please fix this and actually test your change before continuing (please run the program and test it)